search-ui/src/interfaces/corp-type.ts

```diff
@@ -0,0 +1,403 @@
+/**
+ * Corp type codes, used as keys to get corp info.
+ * (sorted by code)
+ */
+ export enum CorpTypeCd {
+    EXTRA_PRO_A = 'A',
+    EXTRA_PRO_B = 'B',
+    BC_COMPANY = 'BC',
+    BENEFIT_COMPANY = 'BEN',
+    CONTINUE_IN = 'C',
+    BC_CCC = 'CC',
+    CCC_CONTINUE_IN = 'CCC',
+    CEMETARY = 'CEM',
+    COOP = 'CP',
+    BC_CORPORATION = 'CR', // SPECIAL NAMEREQUEST-ONLY ENTITY TYPE
+    CONT_IN_SOCIETY = 'CS',
+    ULC_CONTINUE_IN = 'CUL',
+    EXTRA_PRO_REG = 'EPR',
+    FINANCIAL = 'FI',
+    FOREIGN = 'FOR',
+    NR_SOLE_PROP = 'FR', // SPECIAL NAMEREQUEST-ONLY ENTITY TYPE
+    PARTNERSHIP = 'GP',
+    LIBRARY = 'LIB',
+    LICENSED = 'LIC',
+    LL_PARTNERSHIP = 'LL',
+    LIMITED_CO = 'LLC',
+    LIM_PARTNERSHIP = 'LP',
+    MISC_FIRM = 'MF',
+    PRIVATE_ACT = 'PA',
+    PARISHES = 'PAR',
+    PENSION_FUND_SOCIETY = 'PFS',
+    CO_1860 = 'QA',
+    CO_1862 = 'QB',
+    CO_1878 = 'QC',
+    CO_1890 = 'QD',
+    CO_1897 = 'QE',
+    REGISTRATION = 'REG',
+    RAILWAYS = 'RLY',
+    SOCIETY = 'S',
+    SOCIETY_BRANCH = 'SB',
+    SOLE_PROP = 'SP',
+    TRUST = 'T',
+    TRAMWAYS = 'TMY',
+    BC_UNLIMITED = 'UL', // SPECIAL NAMEREQUEST-ONLY ENTITY TYPE
+    BC_ULC_COMPANY = 'ULC',
+    ULC_CO_1860 = 'UQA',
+    ULC_CO_1862 = 'UQB',
+    ULC_CO_1878 = 'UQC',
+    ULC_CO_1890 = 'UQD',
+    ULC_CO_1897 = 'UQE',
+    XPRO_COOP= 'XCP',
+    XPRO_LL_PARTNR = 'XL',
+    XPRO_LIM_PARTNR = 'XP',
+    XPRO_SOCIETY = 'XS'
+  }
+  
+  /** Corp classes. */
+  export enum CorpClass {
+    BC = 'BC',
+    OT = 'OT',
+    SOC = 'SOC',
+    XPRO = 'XPRO',
+    FIRM = 'FIRM'
+  }
+  
+  /** Interface for corp info object. */
+  export interface CorpInfoIF {
+    corpTypeCd: CorpTypeCd
+    colinInd: boolean
+    corpClass: CorpClass
+    shortDesc: string
+    fullDesc: string
+    numberedDesc?: string
+  }
+  
+  /** Array of corp info objects. */
+  export const CorpInfoArray: Array<CorpInfoIF> = [
```

Comment:
I usually put things like this in the 'resource' folder since it isn't really an interface. Is it okay to move this in there?